db/migrate/20200326154315_add_trackable_fields_to_user.rb

```diff
@@ -0,0 +1,11 @@
+class AddTrackableFieldsToUser < ActiveRecord::Migration[5.1]
+  def change
+    change_table :users do |t|
+      t.integer  :sign_in_count, default: 0, null: false
+      t.datetime :current_sign_in_at
+      t.datetime :last_sign_in_at
+      t.inet     :current_sign_in_ip
+      t.inet     :last_sign_in_ip
```

Comment:
you can put this fields on the users migration

Reply:
Done!